src/app/campaign-details/campaign-details.component.html

```diff
@@ -0,0 +1,19 @@
+<div class="container">
+    <img class="banner" src={{campaign.bannerUri}} alt="Campaign Banner">
```

Comment:
I think binding `src` using the expression context is a bit more readable in this case https://angular.io/guide/template-syntax#expression-context